src/rabbit_policy.erl

```diff
@@ -170,9 +170,18 @@ list(VHost) ->
 list_formatted(VHost) ->
     order_policies(list0(VHost, fun format/1)).
 
+list_formatted(VHost, Ref, AggregatorPid) ->
+    list0(VHost, fun format/1, Ref, AggregatorPid),
```

Comment:
is there a reason why here there's a specialized function for the mapping, while in other places the mapping is done inline?